From 26fe1f2189479efa8cd0fc21a5f89c80bc6a2ec5 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Wed, 26 Jan 2022 10:52:07 +0200
Subject: [PATCH 19/64] mkimage: s32gen1: Use 'CONFIG_QSPI_BOOT' instead of
 'CONFIG_FLASH_BOOT'

This regression was introduced by
commit 595c01547f8d ("mkimage: s32gen1: Use -n argument for a configuration file")

Issue: ALB-8303
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-s32/Makefile b/arch/arm/mach-s32/Makefile
index 157bd185ae..5d7b7496ac 100644
--- a/arch/arm/mach-s32/Makefile
+++ b/arch/arm/mach-s32/Makefile
@@ -29,7 +29,7 @@ quiet_cmd_cpp_cfg = CFGS    $@
 
 S32GEN1_CONFIG_FILE = $(CONFIG_S32GEN1_CONFIG_FILE:"%"=%)
 
-ifdef CONFIG_FLASH_BOOT
+ifdef CONFIG_QSPI_BOOT
 $(QSPI_PARAMS_FILE): arch/arm/mach-s32/s32gen1_qspiparam FORCE
 	$(Q)$(objtree)/arch/arm/mach-s32/s32gen1_qspiparam \
 		-m $(QSPI_MEMORY) \
-- 
2.17.1

